utils/debug/PointCloudDebug.js

```diff
@@ -10,39 +10,36 @@ function isInHierarchy(elt, hierarchyNode) {
 
 export default {
     initTools(view, layer, datUi) {
+        const update = () => view.notifyChange(layer);
         layer.debugUI = datUi.addFolder(`${layer.id}`);
 
-        layer.debugUI.add(layer, 'sseThreshold').name('SSE threshold')
-            .onChange(() => view.notifyChange(layer));
-        layer.debugUI.add(layer, 'octreeDepthLimit', -1, 20).name('Depth limit')
-            .onChange(() => view.notifyChange(layer));
-        layer.debugUI.add(layer, 'pointBudget', 1, 15000000).name('Max point count')
-            .onChange(() => view.notifyChange(layer));
+        layer.debugUI.add(layer, 'sseThreshold').name('SSE threshold').onChange(update);
+        layer.debugUI.add(layer, 'octreeDepthLimit', -1, 20).name('Depth limit').onChange(update);
+        layer.debugUI.add(layer, 'pointBudget', 1, 15000000).name('Max point count').onChange(update);
         layer.debugUI.add(layer.object3d.position, 'z', -50, 50).name('Z translation').onChange(() => {
             layer.object3d.updateMatrixWorld();
             view.notifyChange(layer);
         });
-        layer.debugUI.add(layer, 'pointSize', 0, 15).name('Point Size')
-            .onChange(() => view.notifyChange(layer));
-        layer.debugUI.add(layer, 'opacity', 0, 1).name('Opacity')
-            .onChange(() => view.notifyChange(layer));
 
         layer.dbgEnableStickyNode = false;
         layer.dbgStickyNode = '';
         layer.dbgDisplayChildren = true;
         layer.dbgDisplayParents = true;
         layer.dbgDisplaybbox = false;
 
+        var styleUI = layer.debugUI.addFolder('Styling');
+        styleUI.add(layer, 'opacity', 0, 1).name('Layer Opacity').onChange(update);
+        styleUI.add(layer, 'pointSize', 0, 15).name('Point Size').onChange(update);
+        styleUI.add(layer.material, 'visible').name('Visible').onChange(update);
```

Comment:
I'd really move it in the parent folder, it's very useful to be able to hide / show layers without having to unfold something, WDYT?

Reply:
no strong opinion. I have just amended the commit to have the following UI layout :

![image](https://user-images.githubusercontent.com/7373521/41094938-4025e3f8-6a50-11e8-8330-3db9198ced42.png)